Source/HTTP/WinHttp/winhttp_provider.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include <winhttp.h>
-#include "winhttp_connection.h"
+//#include "winhttp_connection.h"
```

Comment:
delete?

Reply:
ya should be removed. Was trying to sort out some weird circular dependencies by removing some unneeded header inclusions and moving a few things around